src/features/projectsV2/ProjectsMap/SingleProjectView.tsx

```diff
@@ -36,12 +41,15 @@ const SingleProjectView = ({ mapRef, setIsOnSampleMarker }: Props) => {
       features: singleProject?.sites ?? [],
     };
   }, [projectSlug]);
-  // Zoom to plant location polygon
+  // Zoom to plant location
   useEffect(() => {
-    const isPlantLocationReadyToZoom =
-      selectedPlantLocation && router.isReady && requestedPlantLocation;
+    const canZoomToPolygonPlantLocation =
+      selectedPlantLocation &&
+      router.isReady &&
+      requestedPlantLocation &&
+      !selectedSamplePlantLocation;
     if (
-      isPlantLocationReadyToZoom &&
+      canZoomToPolygonPlantLocation &&
       selectedPlantLocation.geometry.type === 'Polygon'
     ) {
```

Comment:
`selectedPlantLocation.geometry.type === 'Polygon'`

could be part of the `canZoomToPolygonPlantLocation` logic itself.